manifests/opconfig.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Google LLC
+#
```

Comment:
nit: where did this file come from? I don't see us generating it in `presubmit.sh`.

Reply:
Ah.. copy pasta again, sorry.